src/__acceptance_tests__/__ssr_pages__/carousel.tsx

```diff
@@ -5,12 +5,14 @@ const CarouselTest = (): JSX.Element => (
     <ResponsiveLayout>
         <Carousel
             withBullets
+            autoplay={false}
+            initialActiveItem={0}
             items={Array.from({length: 4}, (_, idx) => (
                 <MediaCard
                     headline={<Tag type="promo">Headline</Tag>}
                     title={'Card ' + idx}
                     description="Description"
-                    media={<Image src="tennis.jpg" aspectRatio="16:9" />}
+                    media={<Image src="tennis.jpg" aspectRatio="16:9" loadingFallback={false} />}
```

Comment:
This test was also experimenting instabilities due to some images randomly loading the fallback in ssr. I think this is due to having increased the number of slides, the images take longer to load before the snapshot. Disabling fallbacks seem to work right